x-pack/plugins/fleet/server/telemetry/sender.ts

```diff
@@ -131,10 +144,15 @@ export class TelemetryEventsSender {
 
       queue.clearEvents();
 
-      this.logger.debug(JSON.stringify(events));
+      const eventsWithLicenseInfo = events.map((event) => ({
+        ...event,
+        license: this.licenseInfo?.license,
+      }));
+
+      this.logger.debug(JSON.stringify(eventsWithLicenseInfo));
 
       await this.send(
-        events,
+        eventsWithLicenseInfo,
```

Comment:
I think this sender is only used for sending upgrade events, we should probably add the license info to FleetUsageSender too.
You can see that logged as
```
[2024-05-01T15:13:14.420+02:00][DEBUG][plugins.fleet] Fleet Usage: {"agents_enabled":true,"agents":{"total_enrolled":1,"healthy":0,"unhealthy":0,"offline":1,"inactive":0,"unenrolled":2,"total_all_statuses":3,"updating":0},"fleet_server":{"total_enrolled":1,"healthy":0,"unhealthy":0,"offline":1,"updating":0,"total_all_statuses":1,"num_host_urls":1}}
```

Reply:
Hey, can I ask for a clarification?

Is it correct that the [FleetUsageSender](https://github.com/elastic/kibana/blob/0288bb466403452dad0fe0958da9a2249a473072/x-pack/plugins/fleet/server/services/telemetry/fleet_usage_sender.ts) sends telemetry data through e.g.
https://github.com/elastic/kibana/blob/0288bb466403452dad0fe0958da9a2249a473072/x-pack/plugins/fleet/server/services/telemetry/fleet_usage_sender.ts#L93
So we could add license info to `fleetUsageData`? Just wondering, as the `TelemetryEventsSender` uses a simple POST request to the telemetry URL.